CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 ### Fixes
 
 - [Pull request #379: Ensure multiple autocompletes on one page do not have conflicting id attributes](https://github.com/alphagov/accessible-autocomplete/pull/379)
+- [Pull request #388: Set aria-selected as a string instead of a boolean to avoid being dropped.](https://github.com/alphagov/accessible-autocomplete/pull/388)
```

Comment:
Sorry, one thing I missed, could you move this CHANGELOG entry to the 'unreleased' section under a new 'fixes' heading please? Then I'll get this merged :)

Reply:
Fixed (it's obvious now that you mentioned it)! Sorry for the messy history, though...